connect-inject/consul_sidecar_test.go

```diff
@@ -27,14 +27,14 @@ var (
 // would require a lot of boilerplate to get at the underlying patches that would
 // complicate understanding the tests (which are simple).
 
-// Test that the lifecycle sidecar is as expected.
-func TestLifecycleSidecar_Default(t *testing.T) {
+// Test that the consul sidecar is as expected.
+func TestConsulSidecar_Default(t *testing.T) {
 	handler := Handler{
-		Log:                       hclog.Default().Named("handler"),
-		ImageConsulK8S:            "hashicorp/consul-k8s:9.9.9",
-		LifecycleSidecarResources: lifecycleResources,
+		Log:                    hclog.Default().Named("handler"),
+		ImageConsulK8S:         "hashicorp/consul-k8s:9.9.9",
+		ConsulSidecarResources: lifecycleResources,
```

Comment:
need to rename `lifecycleResources` too

Reply:
Nice thanks for catching this for some reason I overlooked that as a k8s specific thing but that's just the word lifecycle on its own. Renamed!